pipeline/pipeline.go

```diff
@@ -420,6 +422,14 @@ func (p *Pipeline) In(sourceID SourceID, sourceName string, offset int64, bytes
 			// Dead route, never passed here.
 			return EventSeqIDError
 		}
+	case decoder.NGINX_ERROR:
+		_ = event.Root.DecodeString("{}")
+		err := decoder.DecodeNginxError(event.Root, bytes)
+		if err != nil {
+			p.logger.Errorf("wrong nginx error log format offset=%d, length=%d, err=%s, source=%d:%s, cri=%s", offset, length, err.Error(), sourceID, sourceName, bytes)
+			// Dead route, never passed here.
```

Comment:
But it is not `Dead route` :)

Reply:
> this is same logic as in

It is used logger.Fatal instead of logger.Error
https://github.com/ozontech/file.d/blob/760665d898bb87a8be9ad799f8d127748e80ab0e/pipeline/pipeline.go#L421
So in your case it's not a dead route :)